### DISCUSSION TITLE
Problemi flash tasmota su dispositivo Tuya -

### DISCUSSION THREAD

**Discussion by braudoo** [indomus/forum#530] [HUB: Home Assistant]
Salve a tutti, ho acquistato diversi Sensori Tuya porte/finestre **MC08_W1**
basate sul chip **ESP8266** modello **TYLC4**, con l'intento di flashare il fw Tasmota.
Come primo tentativo ho provato ad utilizzare utilizzando l'applicativo Tuya-convert,
che consente di fare il flash via OTA.
Lancio l'applicativo via SSH, e collego lo smartphone alla rete **vtrust-flash**, come
spiegato in diverse guide.
A questo punto metto il dispositivo in modalità pairing (lampeggio del led velocemente),
e premo INVIO per iniziare la procedura di pairing con la rete **vtrust-flash**, ma dopo
alcuni secondi in cui si vede che il sistema prova a connettersi con il dispositivo, questo
smette di lampeggiare.
Di seguito mi dice di visionare il file *.log per verificare cosa non è andato.
Dei 4 file*.log contenuti nella cartello, questo mi sembra quello più indicativo:

_wlan0: interface state ENABLED->DISABLED
wlan0: AP-DISABLED 
wlan0: CTRL-EVENT-TERMINATING 
nl80211: deinit ifname=wlan0 disabled_11b_rates=0
AP closed
Stopping DNSMASQ server...
Attempting to stop wpa_supplicant
Configuring AP interface...
Error: Device for nexthop is not up.
Error: Device for nexthop is not up.
Starting DNSMASQ server...
Starting AP on wlan0...
Configuration file: /dev/stdin
wlan0: Could not connect to kernel driver
Using interface wlan0 with hwaddr dc:a6:32:4c:1e:a2 and ssid "vtrust-flash"
wlan0: interface state UNINITIALIZED->ENABLED
wlan0: AP-ENABLED 
wlan0: AP-STA-CONNECTED d8:0b:9a:10:3e:58
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED d8:0b:9a:10:3e:58
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED d8:0b:9a:10:3e:58
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED d8:0b:9a:10:3e:58
wlan0: AP-STA-CONNECTED d8:0b:9a:10:3e:58
Attempting to stop wpa_supplicant
Configuring AP interface...
RTNETLINK answers: File exists
Starting DNSMASQ server...
Starting AP on wlan0...
Configuration file: /dev/stdin
wlan0: Could not connect to kernel driver
Using interface wlan0 with hwaddr dc:a6:32:4c:1e:a2 and ssid "vtrust-flash"
wlan0: interface state UNINITIALIZED->ENABLED
wlan0: AP-ENABLED 
wlan0: AP-STA-CONNECTED d8:0b:9a:10:3e:58
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED d8:0b:9a:10:3e:58
wlan0: AP-STA-CONNECTED d8:0b:9a:10:3e:58
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-CONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED 8c:aa:b5:e7:86:5f
wlan0: AP-STA-DISCONNECTED d8:0b:9a:10:3e:58_

Chiedo se qualcuno riesce a capire da questo file log cosa non sta andando.

Metodo alternativo per flashare Tasmota sarebbe con USB/TTL, ma in questo
caso ho visto su un forum che la procedura è alquanto macchinosa, quindi se 
fosse possibile via OTA, tanto di guadagnato.

Grazie per l'aiuto.

**Comment by snakuzzo** [comment]
sul problema che hai con tuya convert non saprei, ma non mi pare sia un problema del device.
Quanto al flash via adattatore, non c'è nulla di macchinoso.

**Comment by braudoo** [comment]
volevo capire perchè non andasse via OTA. 
Proverò a collegarmi con un'altro device alla rete vtrust-flash.
